blog/content/post/2020-04-29-trenchboot-uefi-environment.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Installing Trenchboot in UEFI environments
+abstract: This blog post will show you how to install NixOS on UEFI platforms
+          and how to install Trenchboot on them.
+          
+cover: /covers/trenchboot-logo.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2020-04-29
+archives: "2020"
+
+tags:
+  - firmware
+  - security
+  - open-source
+categories:
+  - Firmware
+  - OS Dev
+  - Security
+
+---
+
+If you haven't read previous blog posts about TrenchBoot project, I encourage
+to do so. We have presented our motivation, project's overview concept and
+goals there. If you want to keep up with project development, stay tuned and
+always try to catch up missed articles.
+
+> Those articles will not always be strictly technical. Therefore, if you think
+some topics should be elaborated, feel free to ask us a question.
+
+In this article, I will explain how to install the NixOS on machines with UEFI
+firmware and hwo to install TrenchBoot. The most essential parts of system will
+be also described in details. It's important to understand their role, so you
+will be able to reproduce the environment smoothly and consciously.
```

Comment:
@miczyg1 what about one paragraph mentioning DRTM TCG spec and difference between early lanuch and late lunch as well as explanation that we discuss early launch but overall goal of TB is to implement both.

Reply:
Done